### DISCUSSION TITLE
Is there a way to load credentials by default when loading new chatflow?

### DISCUSSION THREAD

**Discussion by mzmarkib** [FlowiseAI/Flowise#1165] [Q&A]
I downloaded A chat flow and copy it into another flow, using the load chatflow option in the settings. It does load all the nodes from the previous chatflow... but it does not assign the default credentials from the previous chatflow. (Note: both flows are in the same server/instance) 

- is there a way to make it accept the default or any available credentials by default when loading a chatflow?
- is there a way to change some code to make this happen via typescript or whatever?
- is there a way to inject the load file via API?

Thanks in advance.
Appreciate your quick reply.

Best regards

**Comment by HenryHengZJ** [comment]
Hey unfortunately for security reason, we got rid of all credentials when copy paste
